File: frontend-angular/src/app/dto/gestionar-bus/bus-envio/bRutaEnvioDTO.ts
import {BHorarioEnvioDTO} from './bHorarioEnvioDTO';

export class BRutaEnvioDTO {
  constructor(
    public id: string,
    public codigo: string,
    public Horario: BHorarioEnvioDTO,
    public dentroBus: boolean
  ) {}
}

File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestion_buses/bus_envio/bRutaEnvioDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestion_buses.bus_envio;

import lombok.*;

import java.io.Serializable;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class bRutaEnvioDTO implements Serializable {
    UUID id;
    String codigo;
    bHoriarioEnvioDTO horario;
    Boolean dentroBus;
}